packages/api3-voting/contracts/Api3Voting.sol

```diff
@@ -260,11 +260,11 @@ contract Api3Voting is IForwarder, AragonApp {
         require(userAddressToLastNewProposalTimestamp[msg.sender].add(api3Pool.EPOCH_LENGTH()) < now, "API3_HIT_PROPOSAL_COOLDOWN");
         userAddressToLastNewProposalTimestamp[msg.sender] = now;
 
-        uint64 snapshotBlock = getBlockNumber64() - 1; // avoid double voting in this very block
```

Comment:
The comment sounds like the `- 1` prevents double voting (presumably by the proposal maker). I don't see how.